src/enums/statuses.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Enum for task status codes.
+ * @readonly
+ * @enum {number}
+ */
+export const statusCodes = {
```

Comment:
Is this needed?

Reply:
Added TODO to check this. On the first glance this can be deleted as child markets dont need separate status, it will be inherited from parentMarket